packages/ui-stencil/src/components/internal/orama-chat/orama-chat.tsx

```diff
@@ -241,24 +244,30 @@ export class OramaChat {
     return this.messagesContainerRef.scrollHeight > this.messagesContainerRef.clientHeight
   }
 
+
   render() {
     const lastInteraction = chatContext.interactions?.[chatContext.interactions.length - 1]
     const lastInteractionStatus = lastInteraction?.status
+    const hasInteractions = chatContext.interactions?.length > 0
+    const showHeader = this.showHeader && (hasInteractions || !!this.chatTitle)
 
-    // ? Question: Maybe should be a orama-button variant?
     return (
       <Host>
-        {this.showClearChat && (
-          <div class={{ header: true, hidden: chatContext.interactions?.length === 0 }}>
+        <div class={{ header: true, hidden: !showHeader }}>
```

Comment:
Maybe better to avoid rendering this at all in case `showHeader` is set to false